test/lib/game/game_supervisor_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Reactor.GameSupervisorTest do
+  use ExUnit.Case, async: true
+
+  test "can create a game" do
+    %{workers: workers} = Supervisor.count_children(Reactor.GameSupervisor)
```

Comment:
I think it's worth having just to know that my function is calling that function. I know it's a simple call but I think it's worth knowing